osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -31,6 +31,11 @@ public class OsuDifficultyHitObject : DifficultyHitObject
         /// </summary>
         public readonly double StrainTime;
 
+        /// <summary>
+        /// Normalised distance from the start position of the previous <see cref="OsuDifficultyHitObject"/> to the start position of this <see cref="OsuDifficultyHitObject"/>.
+        /// </summary>
+        public double JumpDistance { get; private set; }
```

Comment:
this naming is very confusing, because in theory this is even more "lazy" than `LazyJumpDistance` since it's ignoring sliderbodies on sliders

Reply:
rename to HeadJumpDistance? CircleJumpDistance?

Reply:
i don't like either of those. maybe swapping `JumpDistance` and `LazyJumpDistance` makes more sense. might need more opinions

Reply:
i don't like it cuz it will be confusing for people who already know pp
maybe changing LazyJumpDistance to TailJumpDistance?

Reply:
that's a fair point. `TailJumpDistance` doesn't quite get there IMO, because it should be general to every object type. something that covers the notion of it being lazy and from the end position would be good. `LazyJumpFromEndDistance` would be ideal for me, but i don't mind long variable names that are descriptive